liberty-maven-plugin/src/it/compile-jsp-it/src/test/java/net/wasdev/wlp/maven/test/app/CompileJSPTest.java

```diff
@@ -46,14 +49,14 @@ public void testXmlElements() throws Exception {
         
         // parse input XML Document
         XPath xPath = XPathFactory.newInstance().newXPath();
-        String expression = "/liberty-plugin-config/featureManager";
+        String expression = "/server/featureManager/feature[text()]";        
         NodeList nodes = (NodeList) xPath.compile(expression).evaluate(inputDoc, XPathConstants.NODESET);
-        Assert.assertEquals("Number of <featureManager/> element ==>", 3, nodes.getLength());
-        
-        xPath = XPathFactory.newInstance().newXPath();
-        expression = "/liberty-plugin-config/feature";
-        nodes = (NodeList) xPath.compile(expression).evaluate(inputDoc, XPathConstants.NODESET);
-        Assert.assertEquals("Value of first <feature/> ==>", "ejbLite-3.2", value);
+        Assert.assertEquals("Number of <feature/> elements ==>", 3, nodes.getLength());
+
+        Assert.assertEquals("Value of first <feature/> ==>", "ejbLite-3.2", nodes.item(0).getTextContent()); 
```

Comment:
Depending on the OS and the JDK, I think the order of these could vary. Would be safer to have a collection of the expected values and just check that the values found in the file are contained in that collection.